src/main/webapp/app/exercises/modeling/assess/modeling-assessment.component.ts

```diff
@@ -270,9 +284,27 @@ export class ModelingAssessmentComponent implements AfterViewInit, OnDestroy, On
             elementType: feedback.referenceType! as UMLElementType | UMLRelationshipType,
             score: feedback.credits!,
             feedback: feedback.text || undefined,
+            label: this.calculateLabel(feedback),
+            labelColor: this.calculateLabelColor(feedback),
         }));
         if (this.apollonEditor) {
             this.apollonEditor!.model = this.model;
         }
     }
+
+    private calculateLabel(feedback: any) {
```

Comment:
can we avoid to use `any` in here? i'm just asking :)

Reply:
sadly I have not found a way, as the datatype comes from Apollon and Artemis can't infer what type it is.